content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -15,45 +15,29 @@ aliases:
 related:
   - /influxdb/cloud-dedicated/query-data/execute-queries/troubleshoot/
 list_code_example: >
-  <!-- Import for tests and hide from users.
-  ```python
-  import os
-  ```
-  -->
-  <!--pytest-codeblocks:cont-->
-  ```python
-  from influxdb_client_3 import(InfluxDBClient3,
-                                WriteOptions,
-                                write_client_options)
-
-  # Instantiate batch writing options for the client
-
-  write_options = WriteOptions()
-  wco = write_client_options(write_options=write_options)
-
-  # Instantiate an InfluxDB v3 client
-
-  with InfluxDBClient3(host=f"{{< influxdb/host >}}",
-                        database=f"DATABASE_NAME",
-                        token=f"DATABASE_TOKEN",
-                        write_client_options=wco) as client:
+  
+  #### Example: Writing and querying data
```

Comment:
Thanks for adding this description. The `list_code_example` should only contain a code block, not Markdown content. I think we should move the description inside the python code block as python comments.